src/mlpack/core/data/imputation_methods/median_imputation.hpp

```diff
@@ -15,13 +15,11 @@
 #include <mlpack/prereqs.hpp>
 
 namespace mlpack {
-namespace data {
```

Comment:
If this is removed, should we not remove this from data? I do not see the purpose of removing this

Reply:
Do you mean moving the imputer code to a different directory?  I agree that it could be reorganized for sure.  It might be better to do that as a different reorganization PR.  I can do it, but do you have an idea for how things should move around?

For the documentation, I wanted it to read `mlpack::Imputer` not `mlpack::data::Imputer`.  I'm ok with keeping only the load/save functionality in the `data` namespace.

Reply:
Okay, it is fine, we can reorganise and add more user-friendly functions in a different PR